tests/javascript/repeatable/spec-setup.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * @copyright   Copyright (C) 2005 - 2016 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ * @package     Joomla
+ * @subpackage  JavaScript Tests
+ * @since       3.6
+ * @version     1.0.0
+ */
+
+define(['jquery', 'text!testsRoot/repeatable/fixtures/fixture.html', 'libs/repeatable', 'jasmineJquery'], function ($, fixture) {
+	$('body').append(fixture);
+
+	weready = jasmine.createSpy('weready');
+	prepare_template = jasmine.createSpy('prepare-template');
+	prepare_modal = jasmine.createSpy('prepare-modal');
+	row_add = jasmine.createSpy('row-add');
+	row_remove = jasmine.createSpy('row-remove');
+	value_update = jasmine.createSpy('value-update');
```

Comment:
Rename the Spies to contain Spy keyword (for better readability). E.g. `spy-row-add`.